File: EXERCÍCIOS JAVA E JAVA POO/Aula11_POO/src/aula11_poo/Professor.java
package aula11_poo;
public class Professor extends Pessoa{
    private String especialidade;
    private float salario;
    
    public void receberAumento(){
        System.out.printf(this.nome+" recebeu um aumento de 15%, passando de R$" + this.salario+ " para R$");
        System.out.println(this.salario+=this.salario*.15);       
    }

    public String getEspecialidade() {
        return especialidade;
    }

    public void setEspecialidade(String especialidade) {
        this.especialidade = especialidade;
    }

    public float getSalario() {
        return salario;
    }

    public void setSalario(float salario) {
        this.salario = salario;
    }
    
}

File: CURSO PHP POO/CURSO POO PHP/Aula11/Professor.php
<?php
require_once 'Pessoa.php';
class Professor extends Pessoa{
    private $especialidade;
    private $salario;
    
    public function receberAumento() {
        echo "<p>$this->nome recebeu um aumento de 25% passando de $this->salario para ".$this->salario += $this->salario *.25;
        echo "</br>";
        echo "</br>";
    }
    function getEspecialidade() {
        return $this->especialidade;
    }

    function getSalario() {
        return $this->salario;
    }

    function setEspecialidade($especialidade) {
        $this->especialidade = $especialidade;
    }

    function setSalario($salario) {
        $this->salario = $salario;
    }

}
